code/api/src/modules/user/tests/userMutation.test.js

```diff
@@ -0,0 +1,28 @@
+const chai = require('chai');
+
+const expect = chai.expect;
+const should = chai.should();
+const url = `http://localhost:8000/`;
+const request = require('supertest')(url);
+
+describe('User mutations', () => {
```

Comment:
I like this convention of putting our tests in with our modules/user and sorting my query test versus mutation test